sonar-php-plugin/pom.xml

```diff
@@ -106,8 +120,8 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <minsize>2500000</minsize>
-                  <maxsize>2600000</maxsize>
+                  <minsize>2400000</minsize>
```

Comment:
@pynicolas why did size decreased?

Reply:
Good question. That's because a few dozen classes from guava are not included anymore (with shade minimization). I have to think about it...